gateway/src/resty/sandbox.lua

```diff
@@ -14,7 +14,7 @@ local setmetatable = setmetatable
 local concat = table.concat
 
 local _G = _G
-local _M = {}
+local _M = { debug = false }
```

Comment:
I agree that this produces a lot of unnecessary logs but leaving it like this is confusing because there's no way to enable this logs apart from changing this line of code. Should we at least add a line here that says that you should change this to true if you ever want to debug this module?

Reply:
Yes, I'll mention it in a comment.
I don't see any situation other than development when you'd want to debug this. So it being a temporary code change is not that terrible in my opinion.